miso-web/src/main/webapp/scripts/list_servicerecord.js

```diff
@@ -113,8 +116,13 @@ ListTarget.servicerecord = {
             });
             list += "</ul>";
             return list;
+          } else {
+            return full.attachments
+              .map(function (f) {
+                f.filename;
```

Comment:
This needs to be a `return` statement

Reply:
@djcooke I am not sure what you mean. I thought this a `return` statement.

Reply:
A function written in this form does not return anything unless you explicitly use the `return` keyword